Changelog.md

```diff
@@ -4,6 +4,11 @@ The change log describes what is "Added", "Removed", "Changed" or "Fixed" betwee
 
 ## Unreleased
 
+## 2.0.0
+
+- Remove legacy layer used to support Symfony ^2.0
```

Comment:
I guess this is about `php-translation/common`?

```suggestion
- Remove legacy layer used to support `php-translation/common` ^2.0 and `symfony/translation` "^3.4|^4.2|^5.0"
```

Reply:
It's more about `symfony/translation` v2.7 which was supported in v1.0
In v1.1 we removed support of `symfony/translation` < 3.4 but kept the legacy layer to avoid a BC Break.
In this new release, we removed the legacy code used to deal with `symfony/translation` < 3.4

Reply:
Ok great. Idea to rephraze to "drop support for symfony/translation < 3.4" so it is more explicit?

Reply:
```suggestion
- Drop `symfony/translation` < 3.4 support by removing legacy layer
```